notes/style-guide.md

```diff
@@ -163,29 +175,27 @@ This is very much a work-in-progress and is not exhaustive.
 
 #### Implicit and explicit arguments
 
-* Functions arguments should be implicit if they can "almost always"
+* Function's arguments should be implicit if they can "almost always"
   be inferred. If there are common cases where they cannot be inferred
   then they should be left explicit.
 
 * If there are lots of implicit arguments that are common to a collection
   of proofs they should be extracted by using an anonymous module.
 
-* Implicit of type `Level` and `Set` can be generalised using `variable`.
-  At the moment the policy is *not* to generalise over any other types in
-  order to minimise the amount of information that users have to keep in
-  their head concurrently.
+* Implicit of type `Level` and `Set` can be generalized using the keyword `variable`. At the moment the policy is *not* to generalize over any other types to minimize the amount of information that users have to keep in their head concurrently.
 
 ## Naming conventions
 
 * Names should be descriptive - i.e. given the name of a proof and the
-  module it lives in then users should be able to make a reasonable
-  guess at what it contains.
+  module it lives in, then users should be able to make a reasonable
+  guess at its meaning.
 
 * Terms from other modules should only be renamed to avoid name clashes,
-  otherwise all names should be used as defined.
+  otherwise, all names should be used as defined.
+
+* Datatype names should be capitalized, being its first letter in uppercase and the remaining letters in lowercase.
 
-* Datatype names should be capitalised and function names should be
-  lowercase.
+* Function names should be not capitalized, being all letters in lowercase.
```

Comment:
This isn't quite true. We use camelCase for function names, e.g. `mapMaybe` in `Data.List.Base`

Reply:
If this is the case, it needs to be reformulated.